Manipulator/ServicesManipulator.php

```diff
@@ -27,13 +28,37 @@ class ServicesManipulator
     /**
      * @var string
      */
-    private $template = '    %s:
+    private $yamlTemplate = '    %s:
         class: %s
         arguments: [~, %s, %s]
         tags:
             - { name: sonata.admin, manager_type: %s, group: admin, label: %s }
 ';
 
+    private $emptyXmlServiceDefinition = <<<XML
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+    </services>
+</container>
+XML;
+
+    private $xmlServiceDefinitionTemplate = <<<XML
+    <service id="%s" class="%s">
+            <argument />
+            <argument>%s</argument>
+            <argument>%s</argument>
+
+            <tag name="sonata.admin" manager_type="%s" group="admin" label="%s" />
+        </service>
+        
+    </services>
```

Comment:
to keep line breaks in existing xml-configs, I have to replace the existing `</services>` tag